index.js

```diff
@@ -1,44 +1,58 @@
-const mongoose = require('mongoose');
 
-// Import of the model Recipe from './models/Recipe.model.js'
+const mongoose = require('mongoose');
 const Recipe = require('./models/Recipe.model');
-// Import of the data from './data.json'
-const data = require('./data');
-
-const MONGODB_URI = 'mongodb://localhost:27017/recipe-app';
-
-//Method 1 : Using Async Await
+const recipesData = require('./data.json'); // Adjust the path as needed
 
-const manageRecipes = async () => {
+async function main() {
   try {
-    // Connection to the database "recipe-app"
-    const dbConnection = await mongoose.connect(MONGODB_URI);
-    console.log(`Connected to the database: "${dbConnection.connection.name}"`);
-
-    // Before adding any recipes to the database, let's remove all existing ones
-    await Recipe.deleteMany();
-
-    // Run your code here, after you have insured that the connection was made
+    // Connect to the database
+    await mongoose.connect('mongodb://127.0.0.1:27017/receipe-app', {
```

Comment:
Aamir, you already had the connection to the database established, there's no need for you to re do it again.